### PR TITLE
[syncBN]

### PR DESCRIPTION
**Pull request by jjsjann123**
test update to resolve
  https://github.com/NVIDIA/apex/issues/134#issue-403525480

Using identical learning rate for both DDP with sync BN and single process BN.
The previous configure leaves the impression that sync BN requires adjusting lr
in the script, which is not true.

### CODE REVIEW

File: tests/synced_batchnorm/two_gpu_unit_test.py [hunk 1]
```diff
@@ -92,6 +92,8 @@ def compare(desc, inp1, inp2, error):
 grad_bn = grad_output_t.clone().detach()
 out_bn = bn(inp_bn)
 out_bn.backward(grad_bn)
+for param in bn.parameters():
+    param.grad = param.grad / args.world_size
```
Comment (mcarilli) [inline tests/synced_batchnorm/two_gpu_unit_test.py]: Maybe add a comment here explaining that the division by world_size is included to make the single-process control case equivalent to the averaging over processes that is done by DDP.
